src/main/java/edu/wisc/my/personalizedredirection/controller/PersonalizedRedirectionController.java

```diff
@@ -67,7 +67,7 @@ public void setRedirectionService(IRedirectionService redirectionService) {
 
             try {
                 url = redirectionService.getUrl(request, dataSource);
-            } catch (PersonalizedRedirectionException e) {
+            } catch (RuntimeException e) {
                 logger.error(
                         "Personalized Redirection Error " + e.getMessage());
```

Comment:
If it's an `error`, I'd prefer a `logger.error("Problem getting the personal URL to redirect to.", e)` so that it goes ahead and logs the full exception with stack trace.